### ISSUE TITLE
Menu does not close with click outside body element

### ISSUE THREAD

**Issue by mrwweb** [joeldbirch/superclick#11]
If using the `<body>` as a wrapping element, the menu fails to close when clicking outside the `<body>` on the page background set by the `<html>` element..

**Comment by joeldbirch** [comment]
Yes, this a known issue. Unfortunately, events only bubble up to body child elements on iOS Safari. I'm not sure if Apple has ever fixed that. I suppose we could add some code so other browsers don't suffer because of this, though. Pull requests are welcome.

**Comment by joeldbirch** [comment]
I heard of a fix for this, but I'm not sure I remember it correctly. It was something like, adding this to your css forces the events to correctly bubble up to the `html` element in iOS Safari:

``` css
html {
  cursor: default;
}
```

Untested.

**Comment by mrwweb** [comment]
Tried both that and `pointer-events: all` on `html` and `body` elements in Chrome with no luck.
